coalib/bearlib/abstractions/Linter.py

```diff
@@ -96,6 +96,9 @@ def _prepare_options(options):
         raise ValueError("Invalid `output_format` specified.")
 
     if options["prerequisite_check_command"]:
+        logging.warn("The bear which uses the following executable --{} "
+                     "uses ``prerequisite_check_command`` which is "
+                     "depreciated.".format(options["executable"]))
         if "prerequisite_check_fail_message" in options:
             assert_right_type(options["prerequisite_check_fail_message"],
                               str,
```

Comment:
For now, we are just giving a deprecation warning https://github.com/coala/coala/pull/2804#issuecomment-251006959.